app/plugins/basic-authentication.md

```diff
@@ -119,6 +121,80 @@ When a client has been authenticated, the plugin will append some headers to the
 
 You can use this information on your side to implement additional logic. You can use the `X-Consumer-ID` value to query the Kong Admin API and retrieve more information about the Consumer.
 
+### Paginate through the basic-auth Credentials
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> This endpoint was introduced in Kong 0.11.2.
+</div>
+
+You can retrieve all the basic-auth Credentials for all the Consumers using
```

Comment:
"for all Consumers" seems more correct to me - don't bother with this, I will manually merge and make a minor edit!